controller/two_lane_queue_test.go

```diff
@@ -75,17 +75,12 @@ func TestDoubleKey(t *testing.T) {
 	select {
 	case <-sentinel:
 		t.Error("The sentinel should not have fired")
-	case <-time.After(200 * time.Millisecond):
+	case <-time.After(500 * time.Millisecond):
 		// Expected.
 	}
 	// This should permit the re-reading of the same key.
 	q.Done(k)
-	select {
-	case <-sentinel:
-		// Expected.
-	case <-time.After(200 * time.Millisecond):
-		t.Error("The item was not processed as expected")
```

Comment:
I don't understand why we'd stop checking for this within a reasonable period of time?

Reply:
Yeah, I think the problem is differnet.